inc/gui.class.php

```diff
@@ -286,7 +286,7 @@ public static function options_page() { ?>
 									$iso_codes_link = sprintf(
 										'<a href="%s" target="_blank" rel="noopener noreferrer">',
 										esc_url(
-											__( 'https://www.iso.org/iso/country_names_and_code_elements', 'antispam-bee' ),
+											__( 'https://www.iso.org/obp/ui/#search', 'antispam-bee' ),
```

Comment:
The proposed link leads to the empty search page when opened outside an existing session and is not really helpful in this case.

Guess the intention was https://www.iso.org/obp/ui/#search/code/
(also see https://github.com/pluginkollektiv/antispam-bee/issues/462#issuecomment-1296804571 for discussion)